.ci/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,146 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+import org.kie.jenkins.MavenCommand
+
+changeAuthor = env.ghprbAuthorRepoGitUrl ? util.getGroup(env.ghprbAuthorRepoGitUrl) : (env.ghprbPullAuthorLogin ?: CHANGE_AUTHOR)
+changeBranch = env.ghprbSourceBranch ?: CHANGE_BRANCH
+changeTarget = env.ghprbTargetBranch ?: CHANGE_TARGET
+
+pipeline {
+    agent {
+        label 'kie-rhel7 && kie-mem16g'
+    }
+    tools {
+        maven 'kie-maven-3.6.3'
+        jdk 'kie-jdk11'
+    }
+    options {
+        timestamps()
+        timeout(time: getTimeoutValue(), unit: 'MINUTES')
+    }
+    environment {
+        SONARCLOUD_TOKEN = credentials('SONARCLOUD_TOKEN')
+        MAVEN_OPTS = '-Xms1024m -Xmx4g'
```

Comment:
Is that useful here ?

Reply:
I checked logs of the job, I didn't found it there, so removed, thank you